codelab-master/apps/playground/src/app/quiz/containers/quiz-selection/quiz-selection.component.html

```diff
@@ -0,0 +1,88 @@
+<ng-container *ngIf="quizData.length > 0">
```

Comment:
At what point would we pass quizData with length 0?
Is this even a valid use case? If not, let's just fail fast.
This way the user of this component would get an error message, not an empty screen

Reply:
got it